lib/bundle/locker.rb

```diff
@@ -7,10 +7,12 @@
 
 module Bundle
   module Locker
+    LOCKFILE_FORMAT_VERSION = "1"
+
     module_function
 
-    def lockfile(global: false, file: nil)
-      brew_file_path = Brewfile.path(global: global, file: file)
+    def resolve_lockfile_path(global: false, brewfile: nil)
```

Comment:
I think the older name makes more sense with what we use elsewhere for file paths.

Reply:
I want the caller to understand that there's a resolution process behind this method based on its parameters because of the call to `Brewfile#path`.

Reply:
@colindean this sort of refactoring is better done in another PR where it's made consistent with all naming across homebrew/bundle. Inconsistency across the repository is worse than having the name here be suboptimal, IMO.